source/installation/with/yunohost.html.md

```diff
@@ -2,8 +2,19 @@
 title: Installing with YunoHost
 ---
 
-This installation method is not yet documented. Your help would be welcome!
+You must have a Yunohost server.
 
-TODO: check this link is working
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.
 
-<a href="https://contribute.joinplu.me/#/documentation">Help writing this page</a>
\ No newline at end of file
+You can install it from the [Yunohost Apps](https://yunohost.org/#/apps_en), select All apps in the dropdown menu, Search for Plume, click on install and follow the steps
+
+or
+
+using from an SSH connexion to your Yunohost server: 
+```bash 
+sudo yunohost app install plume
+
+or 
```

Comment:
i would make this **two** `bash` sections, and leave the "or" out of it.

Reply:
perhaps briefly explain what the difference is between the two methods:

"or, if you prefer to go bleeding edge,"

Reply:
I removed the first way to install using SSH as the prerequisite is to add the community app list in Yunohost. But in a few weeks, community and official lists will be merged in a single list. So I let the universal way to install it.